webapp/src/components/ShoppingCart.tsx

```diff
@@ -31,98 +31,109 @@ function ShoppingCart(props: ShoppingCartProps): JSX.Element {
-                              props.onIncrementUnit(cartItem.product)
-                            }
-                            disabled={!handleButton(cartItem)}
-                            className="m-1"
-                          >
-                            +
-                          </Button>
-                        </Stack>
-                      </TableCell>
-                      <TableCell>
-                        <Typography component="div">
-                          {cartItem.product.price}€
-                        </Typography>
-                      </TableCell>
-                    </TableRow>
-                  );
-              })}
-            </TableBody>
-          </Table>
-        </TableContainer>
-        <Typography variant="h6" className="m-2">
-          Total Price -{" "}
-          {
-            calculateTotal(props.products, 0) // There are no shipping costs yet here
-          }
-          €
-        </Typography>
-      </React.Fragment>
-    );
-  else
-    return (
-      <Typography variant="h6" className="m-2">
-        Shopping cart is empty :(
-      </Typography>
-    );
+    const Img = styled("img")({
+        display: "block",
+        width: "25%",
+    });
+
+    if (props.totalUnitsInCart > 0)
+        return (
+            <React.Fragment>
+                <TableContainer component={Paper}>
+                    <Table>
+                        <TableHead>
+                            <TableRow>
+                                <TableCell> Product </TableCell>
+                                <TableCell> Quantity </TableCell>
+                                <TableCell> Price per unit </TableCell>
+                            </TableRow>
+                        </TableHead>
+                        <TableBody>
+                            {props.products.map((cartItem: CartItem) => {
+                                if (cartItem.amount > 0)
+                                    return (
+                                        <TableRow key={cartItem.product.code}>
+                                            <TableCell>
+                                                <Stack
+                                                    direction={{xs: "column", sm: "row"}}
+                                                    spacing={{xs: 1, sm: 2, md: 4}}
+                                                    justifyContent="flex-start"
+                                                    alignItems="center"
+                                                >
+                                                    <Img
+                                                        alt="Imagen del producto en el carrito"
+                                                        src={require("../images/"
+                                                            .concat(cartItem.product.code)
+                                                            .concat(".jpg"))}
```

Comment:
Check this, because now the image products are .png

Reply:
Oh that's true! My bad! 👌